lib/containers/container_images.pm

```diff
@@ -95,7 +95,7 @@ sub test_opensuse_based_image {
 
     my $distri  = $args{distri}  //= get_required_var("DISTRI");
     my $version = $args{version} //= get_required_var("VERSION");
-
+    my $host_os = $args{host_os};
```

Comment:
why do you need this?  I don't see you use this variable in this function.

Reply:
forgotten. has been removed. i was thinking to add some logic here which i did not need at the end